addons/mail/static/src/components/message/message.xml

```diff
@@ -186,9 +186,9 @@
                                 </div>
                             </div>
                             <div
-                                t-if="messageView.messageActionList and !messageView.composerViewInEditing"
-                                class="o_Message_actionListContainer"
-                                t-att-class="{
+                                    t-if="messageView.messageActionList and !messageView.composerViewInEditing"
+                                    class="o_Message_actionListContainer"
+                                    t-att-class="{
```

Comment:
Don't know why all this diff. If it's you text editor that does it automatically when saving changes, you should disable this feature. Otherwise you will make lots of useless diff.